src/app.rs

```diff
@@ -147,6 +145,21 @@ impl App {
         self.zoom -= self.zoom_factor;
     }
 
+    pub fn move_pose_estimate(&mut self, x: f64, y: f64, yaw: f64) {
+        self.pose_estimate.2 += yaw;
```

Comment:
minimal should be a struct. it would make the coding much clearer. This would also allow some methods for it